src/Core/src/Handlers/TimePicker/TimePickerHandler.iOS.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using Foundation;
+using UIKit;
+using RectangleF = CoreGraphics.CGRect;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class TimePickerHandler : AbstractViewHandler<ITimePicker, MauiTimePicker>
+	{
+		MauiTimePicker? _nativeTimePicker;
+		static UIDatePicker? Picker;
```

Comment:
Why is this static? When we want to map the stuff, we are already receiving the handler. So we can just use that.

Reply:
Yeah, this is ... wrong. I'm rewriting this now.